vint/__main__.py

```diff
@@ -0,0 +1,15 @@
+from vint.bootstrap import (
+    init_logger,
+    init_linter,
+    init_cli,
+)
+
+
+def main():
+    init_logger()
+    init_linter()
+    init_cli()
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
Why not use `vint.main()` (as with the entry point)?

Reply:
Ping?

Reply:
No idea why I didnt do that ;-)

All green now again.

Reply:
Great, thanks!